temba/msgs/tests.py

```diff
@@ -2103,3 +2107,120 @@ def test_render(self):
         # exception if tag not used correctly
         self.assertRaises(ValueError, self.render_template, '{% load sms %}{% render with bob %}{% endrender %}')
         self.assertRaises(ValueError, self.render_template, '{% load sms %}{% render as %}{% endrender %}')
+
+
+class CeleryTaskTest(TransactionTestCase):
```

Comment:
@nicpottier Please have a look now. `TembaTest` actually inherits from `TransactionTestCase` a few levels up, so I just had to disable the rollback things.